README.md

```diff
@@ -19,3 +20,6 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The runtime complexity of the brute-force sorting algorithm, which generates all permutations of the input list, is $\ O(n!)$ in the worst case, as there are $n!$ Possible permutations. For each permutation, we check and compare if it’s sorted and in the correct order, which takes $\ O(n)$ time. Therefore, the overall time complexity for my brute-sort-algorithm, for the checking each permutation, is $\ O(n!)$ in the worst case. The best-case input is an already sorted list, which results in a best-case time complexity of $\ O(n)$. If we randomly created permutations without memory, I believe the worst-case time complexity would remain $\ O(n!)$, but the specific number of permutations attempted before finding the sorted list could vary. However it wouldn’t change the overall complexity. 
```

Comment:
You're still not accounting for the time it takes to check whether a permutation is sorted.

Reply:
If I'm getting it correct,  would that be O(n! * n) as giving it's worst case?

Reply:
Yes.